platform/core.network/test/unit/src/org/netbeans/core/network/utils/LocalAddressUtilsTest.java

```diff
@@ -1,82 +1,96 @@
 /*
```

Comment:
Are you the author of this file? If not, please revert the license change. I think it was introduced by @lbruun and I assume he would be ok with the change, but we should be correct.

Reply:
Reverted